### PR TITLE
WOE 13 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/specter_of_mortality.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Name:Specter of Mortality
+ManaCost:3 B B
+Types:Creature Specter
+PT:3/3
+K:Flying
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, you may exile one or more creature cards from your graveyard. When you do, each other creature gets -X/-X until end of turn, where X is the number of cards exiled this way.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | ChangeType$ Creature.YouOwn | Hidden$ True | Destination$ Exile | Optional$ You | ChangeNum$ X | SelectPrompt$ Select one or more creature cards from your graveyard | SubAbility$ DBImmediateTrig | RememberChanged$ True | AITgtOwnCards$ True | AIMinTgts$ 3
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ Remembered | ConditionCompare$ GE1 | SubAbility$ DBCleanup | Execute$ TrigPumpAll | TriggerDescription$ When you discard a card this way, destroy target nonland permanent an opponent controls with mana value less than or equal to the mana value of the discarded card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/specter_of_mortality.txt:8]: why remember?

File: forge-gui/res/cardsfolder/upcoming/spel_stutter.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Name:Spell Stutter
+ManaCost:1 U
+Types:Instant
+A:SP$ Counter | TargetType$ Spell | ValidTgts$ Card | UnlessCost$ X | SpellDescription$Counter target spell unless its controller pays {2} plus an additional {1} for each Faerie you control.
+SVar:X:Count$Valid Faerie.YouCtrl /Plus.2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/spel_stutter.txt:5]: spacing

File: forge-gui/res/cardsfolder/upcoming/succumb_to_the_cold.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Succumb to the Cold
+ManaCost:2 U
+Types:Instant
+A:SP$ Tap | ValidTgts$ Creature.OppCtrl | SubAbility$ DBCounter | TargetMin$ 0 | TargetMax$ 2 | TgtPrompt$ Select up to one or two target creatures an opponent controls | SpellDescription$ Tap one or two target creatures an opponent controls. Put a stun counter on each of them. (If a permanent with a stun counter would become untapped, remove one from it instead.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/succumb_to_the_cold.txt:4]: min wrong

**Comment by Simisays** [comment]
seems like Scepter of Morality's debuff effect got messed up when I converted it to an immediatetrigger.

I think SVar:Y:Count$TriggerRememberAmount is not correct ?

File: forge-gui/res/cardsfolder/upcoming/specter_of_mortality.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ManaCost:3 B B
+Types:Creature Specter
+PT:3/3
+K:Flying
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, you may exile one or more creature cards from your graveyard. When you do, each other creature gets -X/-X until end of turn, where X is the number of cards exiled this way.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | ChangeType$ Creature.YouOwn | Hidden$ True | Destination$ Exile | Optional$ You | ChangeNum$ X | SelectPrompt$ Select one or more creature cards from your graveyard | SubAbility$ DBImmediateTrig | RememberChanged$ True | AITgtOwnCards$ True | AIMinTgts$ 3
+SVar:DBImmediateTrig:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ Remembered | ConditionCompare$ GE1 | SubAbility$ DBCleanup | Execute$ TrigPumpAll | TriggerDescription$ When you do, each other creature gets -X/-X until end of turn, where X is the number of cards exiled this way.
+SVar:TrigPumpAll:DB$ PumpAll | ValidCards$ Creature.Other | NumAtt$ -Y | NumDef$ -Y
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:Y:Count$TriggerRememberAmount
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/specter_of_mortality.txt:11]: TriggerRemembered$Amount

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/specter_of_mortality.txt:11]: thanks working now

File: forge-gui/res/cardsfolder/upcoming/stonesplitter_bolt.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Stonesplitter Bolt
+ManaCost:X R
+Types:Instant
+K:Bargain
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | NumDmg$ Z | SpellDescription$ CARDNAME deals X damage to target creature or planeswalker. If this spell was bargained, it deals twice X damage to that permanent instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/stonesplitter_bolt.txt:5]: wrong tgts

**Pull request merged**